.swiftlint.yml

```diff
@@ -10,3 +10,4 @@ disabled_rules:
   - type_body_length
   - function_body_length
   - large_tuple
+  - identifier_name
```

Comment:
@ianpartridge I'm curious - why did this rule start triggering an error in some cases rather than a warning, and only on the xcode9.3beta build?  Is this a temporary measure?

Reply:
Different version of SwiftLint, for some reason.

Reply:
Found it: https://github.com/realm/SwiftLint/issues/2134